Tests/Tests_GCDWheel.cs

```diff
@@ -97,19 +98,18 @@ public void Test_mudra_double_queue() {
         }
         [TestMethod]
         public void Test_exact_slidegcd_end() {
-            GCDWheel wheel = new(null) {
-                ogcds = new()
-                {
-                    { 0, new AbilityTiming(0.35f, false) },
-                    { 0.5f, new AbilityTiming(0.64f, false) }
-                },
-                TotalGCD = 0.5f
-            };
-            wheel.SlideGCDs(0.5f, true);
+            var abilityManager = AbilityManager.Instance;
+            abilityManager.UpdateOGCDs(new Dictionary<float, AbilityManager.AbilityTiming>
+            {
+                { 0, new AbilityManager.AbilityTiming(0.35f, false) },
+                { 0.5f, new AbilityManager.AbilityTiming(0.64f, false) }
+            });
+            var helper = new GCDHelper(null, null);
```

Comment:
Missing the line setting the GCD to 0.5f.
`helper.TotalGCD = 0.5f;`
This is because this test is basically created to test mudras work correctly.

Reply:
Done.